charts/redpanda/values.go

```diff
@@ -833,7 +833,8 @@ type Sidecars struct {
 		DecommissionRequeueTimeout string `json:"decommissionRequeueTimeout"`
 	} `json:"brokerDecommissioner"`
 	ConfigWatcher struct {
-		Enabled bool `json:"enabled"`
+		Enabled                      bool `json:"enabled"`
+		SkipClusterConfigurationSync bool `json:"skipClusterConfigurationSync"`
```

Comment:
In another PR, I  [added a hidden `Args` field on the sidecar itself](https://github.com/redpanda-data/redpanda-operator/pull/937/files#diff-df34075ea48d6120c348021a3727de5f513afd338dfa2e8378993b5b1f374d39R672-R674). As this isn't something that users are going to control, what would you think about using the same field? You can make the same change here and one of us will have a small merge conflict to fix.